test/common_tester.h

```diff
@@ -537,6 +539,14 @@ class CommonTester : public ::testing::Test
     );
   }
 
+  void evalSmallAdj()
+  {
+    const Tangent delta_other = Tangent::Random();
+
+    EXPECT_EIGEN_NEAR((delta.smallAdj() * delta_other).hat(),
```

Comment:
What is the operator '*' doing in this context? 
  - Is it matrix product? Of so, what is `hat()` doing for the case of `Rn`? Because in `Rn` we said `v.hat() = v`
  - If it is not matrix product, what is it? Group composition? If so, what is `hat()` doing? Returning an element of the group? This should not be so.

Reply:
This is a matrix product and 
```cpp
rn^ = | 0 t |
      | 0 0 |
```